Joknierm patch 1 (#8837)
MicrosoftDocs/windows-itpro-docs: windows/client-management/mdm/assignedaccess-csp.md

<a href="" id="assignedaccess-configuration"></a>**./Device/Vendor/MSFT/AssignedAccess/Configuration**
Added in Windows 10, version 1709. Specifies the settings that you can configure in the kiosk or device. This node accepts an AssignedAccessConfiguration xml as input to configure the device experience. For details about the configuration settings in the XML, see [Create a Windows 10 kiosk that runs multiple apps](https://docs.microsoft.com/windows/configuration/lock-down-windows-10-to-specific-apps). Here is the schema for the [AssignedAccessConfiguration](#assignedaccessconfiguration-xsd).

> Updated in Windows 10, version 1909. Added Microsoft Edge kiosk mode. This allows Microsoft Edge to be the specified kiosk application. For details about configuring Microsoft Edge kiosk mode, see [Configure a Windows 10 kiosk that runs Microsoft Edge](https://docs.microsoft.com/DeployEdge/microsoft-edge-configure-kiosk-mode). Windows 10, version 1909 also allows for configuration of the breakout sequence. The breakout sequence specifies the keyboard shortcut that returns a kiosk session to the lock screen.

Reviewer: - When the OS update is released with the DCR change in, we should add a note here and link it to the update KB article, so customers know which update they need to apply.
- Shall we say "Added Microsoft Edge kiosk mode support"?
- Can we add some more descriptions here for the breakout key sequence customization, e.g. the format it accepts, a few samples?

Author: Changed to "Added Microsoft Edge kiosk mode support". Added in format for breakout sequence with an example if you think more examples in this section would be beneficial let me know.

<a href="" id="assignedaccess-configuration"></a>**./Device/Vendor/MSFT/AssignedAccess/Configuration**
Added in Windows 10, version 1709. Specifies the settings that you can configure in the kiosk or device. This node accepts an AssignedAccessConfiguration xml as input to configure the device experience. For details about the configuration settings in the XML, see [Create a Windows 10 kiosk that runs multiple apps](/windows/configuration/lock-down-windows-10-to-specific-apps). Here is the schema for the [AssignedAccessConfiguration](#assignedaccessconfiguration-xsd).

> Updated in Windows 10, version 1909. Added Microsoft Edge kiosk mode support. This allows Microsoft Edge to be the specified kiosk application. For details about configuring Microsoft Edge kiosk mode, see [Configure a Windows 10 kiosk that runs Microsoft Edge](https://docs.microsoft.com/DeployEdge/microsoft-edge-configure-kiosk-mode). Windows 10, version 1909 also allows for configuration of the breakout sequence. The breakout sequence specifies the keyboard shortcut that returns a kiosk session to the lock screen. The breakout sequence is defined with the format modifiers + keys. An example breakout sequence would look something like "shift+alt+a", where "shift" and "alt" are the modifiers and "a" is the key.

Reviewer 2: ```suggestion
Updated in Windows 10, version 1909. Added Microsoft Edge kiosk mode support. This allows Microsoft Edge to be the specified kiosk application. For details about configuring Microsoft Edge kiosk mode, see [Configure a Windows 10 kiosk that runs Microsoft Edge](/DeployEdge/microsoft-edge-configure-kiosk-mode). Windows 10, version 1909 also allows for configuration of the breakout sequence. The breakout sequence specifies the keyboard shortcut that returns a kiosk session to the lock screen. The breakout sequence is defined with the format modifiers + keys. An example breakout sequence would look something like "shift+alt+a", where "shift" and "alt" are the modifiers and "a" is the key.
```

Author: made recommended change

## AssignedAccessConfiguration XSD

> Below schema is for AssignedAccess Configuration up to Windows 10 1909 release.

Reviewer: Shall we say "up to Windows 10 20H2 release" as it is the latest Windows 10 release in the market? https://docs.microsoft.com/en-us/windows/release-information/

Reviewer 3: Maybe that is correct, I haven't checked yet. Here is a suggestion you can use to commit the change (unless you want to edit the file manually to add more material):
```suggestion
The schema below is for AssignedAccess Configuration up to Windows 10 20H2 release.
```
I also find it more natural to say "The schema below" than "Below schema", but that could be caused by being used to European/British English usage.

Author: Added recommended changes.

> The schema below is for AssignedAccess Configuration up to Windows 10 20H2 release.

Reviewer 2: ```suggestion
The following schema is for AssignedAccess Configuration up to Windows 10 20H2 release.
```

</xs:schema>
```

> Schema for features introduced in Windows 10, version 1909.

Reviewer: Can we describe the features? Something like "Here is the schema for features introduced in Windows 10, version 1909 to support Microsoft Edge kiosk mode and breakout key sequence customization.

Author: Update comment to specify the specific changes present in the schema.

## Example AssignedAccessConfiguration XML

> Example XML configuration for a multi app kiosk.

Reviewer 3: 2 cosmetic/codestyle details here: hyphen between multi and app, colon to indicate that the described item follows:
```suggestion
Example XML configuration for a multi-app kiosk:
```

Author: added hyphen and colon

</AssignedAccessConfiguration>
```

Example XML configuration for a Microsoft Edge kiosk. This Edge kiosk s configure to launch www.bing.com on startup.
```xml
<?xml version="1.0" encoding="utf-8" ?>
<AssignedAccessConfiguration
  xmlns="http://schemas.microsoft.com/AssignedAccess/2017/config"
  xmlns:v4="http://schemas.microsoft.com/AssignedAccess/2021/config"
  >
  <Profiles>
    <Profile Id="{AFF9DA33-AE89-4039-B646-3A5706E92957}">
      <KioskModeApp v4:ClassicAppPath="%ProgramFiles(x86)%\Microsoft\Edge\Application\msedge.exe --no-first-run --kiosk www.bing.com"/>
    </Profile>
  </Profiles>
  <Configs>
    <Config>
      <Account>EdgeKioskUser</Account>
      <DefaultProfile Id="{AFF9DA33-AE89-4039-B646-3A5706E92957}"/>
    </Config>
  </Configs>
</AssignedAccessConfiguration>
```

> Example XML configuration for setting a breakout sequence to be Shift+Windows on a Microsoft Edge kiosk. Note BreakoutSequence can be applied to any kiosk type not just an Edge kiosk.

Reviewer 3: Depending on how important the note is, it could be implemented as a colored Note blob. Otherwise, a colon should be used:
```suggestion
Example XML configuration for setting a breakout sequence to be Shift+Windows on a Microsoft Edge kiosk. Note: **BreakoutSequence** can be applied to any kiosk type, not just an Edge kiosk.
```
Also: it is preferable to use some form of highlighting (**bold** or _italics_), or just code marking ( ` ` ) for a keyword.
Added a natural readability comma.

Author: Added note blob, bolded the xml keyword

</AssignedAccessConfiguration>
```

> Example XML configuration for a Microsoft Edge kiosk. This Edge kiosk s configure to launch www.bing.com on startup.

Reviewer: The 2nd sentence seems incomplete. Shall we say "This Microsoft Edge kiosk is configured to launch www.bing.com on startup in a public browsing mode", and also update the below two XML samples to include the kiosk mode type command line.

Reviewer 3: Sounds good to me. Here is a suggestion, in case you prefer to use the "commit suggestion" feature to add more material:
```suggestion
Example XML configuration for a Microsoft Edge kiosk. This Microsoft Edge kiosk is configured to launch www.bing.com on startup in a public browsing mode.
```

```xml
<?xml version="1.0" encoding="utf-8" ?>
<AssignedAccessConfiguration
  xmlns="http://schemas.microsoft.com/AssignedAccess/2017/config"
  xmlns:v4="http://schemas.microsoft.com/AssignedAccess/2021/config"
  >
  <Profiles>
    <Profile Id="{AFF9DA33-AE89-4039-B646-3A5706E92957}">
      <KioskModeApp v4:ClassicAppPath="%ProgramFiles(x86)%\Microsoft\Edge\Application\msedge.exe"
                                  v4:ClassicAppArguments="--no-first-run --kiosk-idle-timeout-minutes=5 --kiosk www.bing.com"/>
    </Profile>
  </Profiles>
  <Configs>
    <Config>
      <Account>EdgeKioskUser</Account>
      <DefaultProfile Id="{AFF9DA33-AE89-4039-B646-3A5706E92957}"/>
    </Config>
  </Configs>
</AssignedAccessConfiguration>
```

Example XML configuration for setting a breakout sequence to be Ctrl+A on a Microsoft Edge kiosk.

> >[!Note]
> **BreakoutSequence** can be applied to any kiosk type, not just an Edge kiosk.

Reviewer 3: To make sure this Note blob is kept safe from formatting on previous lines and also kept from interfering with succeeding lines, the recommended format looks like this:
```suggestion

> [!NOTE]
> **BreakoutSequence** can be applied to any kiosk type, not just an Edge kiosk.

```
Notice that I have put the Note blob keyword "NOTE" in all caps, which is the prevalent format of MarkDown Note blob keywords here in the MicrosoftDocs repositories.

Author: Changed note keyword to all caps
